evap/static/ts/src/slider.ts

```diff
@@ -15,10 +15,8 @@ export class RangeSlider {
     private readonly maxLabel: HTMLSpanElement;
     private readonly minLabel: HTMLSpanElement;
     private readonly rangeLabel: HTMLSpanElement;
-    private readonly min = 0;
-    private max = 0;
-    private low = 0;
-    private high = 0;
+    private allowed: Range = { low: 0, high: 0 };
+    private _selection: Range = { low: 0, high: 0 };
```

Comment:
I now wonder whether `value` would be a better name, to be consistent with other inputs :thinking: